tests/test_construction.py

```diff
@@ -353,6 +353,18 @@ class PickleModelTwo(BaseModel):
     assert not hasattr(m3, '_foo_')
 
 
+def test_pickle_root_model(create_module):
```

Comment:
Maybe it would be better to keep consolidating `RootModel` tests in `tests/test_root_model.py`?